index.js

```diff
@@ -249,13 +249,7 @@ const fillFieldWithTextWhenVisibleAndPressEnter = function(
 ) {
   this.fillFieldWithTextWhenVisible(
     htmlElement,
-    text,
-    timeoutInMilliseconds,
-    errorMessage
-  );
-  this.fillFieldWithTextWhenVisible(
-    htmlElement,
-    protractor.Key.ENTER,
+    text + protractor.Key.ENTER,
```

Comment:
I prefer not making this change and leave it explicit that in the first piece of code the text is being sent and in the second part that the ENTER key press is sent. The performance gain is to low anyway.

Reply:
I think it's an unnecessary complexity, is not it?

Reply:
I'll have to test your proposal first.

Reply:
Ok, this works fine and it really simplify stuff. Good job.